test-violations.py

```diff
@@ -0,0 +1,62 @@
+from attrs import define
+from attrs import field
+
+
+@attr.s(auto_attribs=True)
+class ExampleClass(object):
+    # ruleid:attr-mutable-initializer
+    empty_dict = {}
```

Comment:
:exclamation:**Cycode**: SAST violation: 'Unsafe usage of mutable initializer with attr.s decorator'.
Severity: Medium <!-- 99dcc86a-d0cb-4f58-ba8d-8ef362f87e68 -->